selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/ImportProjectFromGitHubTest.java

```diff
@@ -81,6 +85,7 @@ public void checkAbilityImportProjectFromGithub() throws Exception {
     dashboard.selectWorkspacesItemOnDashboard();
     workspaces.clickOnAddWorkspaceBtn();
     newWorkspace.waitToolbar();
+    newWorkspace.clickOnAllStacksTab();
```

Comment:
please, leave comment about why we can't rely on Quick Start stack list.

Reply:
The **Java** stack, which we use for creating test workspaces, not exist **Quick Start** stack list on **OSIO**.